### DISCUSSION TITLE
Enable variables in response filter and ability to use JQ as an option

### DISCUSSION THREAD

**Discussion by admodras** [Kong/insomnia#3686] [Ideas]
**Is your feature request related to a problem? Please describe.**
There are several requests that are very repeatable and sometimes there is a very large response body. 

Having the ability to set up environment variables - like {{ response.fault }} to always filter out just the faults from the response body.  Currently there is no way to utilize variables in the jsonPATH filter section of Insomnia. 

**Describe the solution you'd like**
Most desirable outcome would be to be able to switch between JQ and jsonPATH and then having the ability to save the appropriate filter to an environment variable,  so we can have multiple different responses filtered based on environment or request.

**Comment by dimitropoulos** [comment]
> to be able to switch between

Are you saying you'd like to use _both_ jq and jsonpath?

Can you please provide an example of what you're trying to do with mockbin.org or something?  I'd like to understand your current workflow compared to the desired workflow.
